onnx/test/reference_evaluator_save_test.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+# pylint: disable=C3001,C0302,C0415,R0904,R0913,R0914,R0915,W0221,W0707
```

Comment:
```suggestion
```

prefer disabling inline instead so we don’t disable everything for the whole module